website/blog/2023-10-17-seed-round-release-0-3-0-RAG.md

```diff
@@ -0,0 +1,63 @@
+---
+authors: [ meng, gyxlucy ]
+
+tags: [release]
+
+---
+# Announcing our $3.2M seed round, and the long-awaited RAG release in Tabby v0.3.0
```

Comment:
consider this is the blog for `tabby`, let move this announcement to the bottom of blog post?

Reply:
Actually this blog is more about fundraising so let's keep it this way.